cli/tests/integration_tests.rs

```diff
@@ -1,5 +1,8 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-use crate::colors::strip_ansi_codes;
+#[macro_use]
+extern crate log;
+
+use deno_cli::colors::strip_ansi_codes;
```

Comment:
Just curious, why the change from `crate::colors` to `deno_cli::colors`?
It's still the same crate right?

Reply:
No, this is another crate that gets built after deno_cli (crucially that's when the deno binary is available)